app/src/main/java/com/kickstarter/libs/utils/SocialUtils.java

```diff
@@ -131,12 +133,25 @@ public static String friendBackingActivityTitle(final @NonNull Context context,
   /**
    * Returns a namepile for a list of friends.
    */
-  public static @NonNull String projectCardFriendNamepile(final @NonNull List<User> friends, final @NonNull KSString ksString) {
-    final String friendName = friends.size() >= 1 ? friends.get(0).name() : "";
-    final String secondFriendName = friends.size() >= 2 ? friends.get(1).name() : "";
-    final String remainingCount = NumberUtils.format(Math.max(0, friends.size() - 2));
+  public static @NonNull String projectCardFriendNamepile(final Context context, final @NonNull List<User> friends, final @NonNull KSString ksString) {
+    final String friendName;
+    if (friends.size() < 3) {
+      friendName = friends.size() >= 1 ? friends.get(0).name() : "";
+    } else {
```

Comment:
this logic makes sense, but is a little hard to read through. maybe it'd be helpful to leave a lil comment here with an example of what `friendName` would be set here